docs/proposals/2019-03-18-Links-List.md

```diff
@@ -0,0 +1,112 @@
+## Problem
+
+Organizations have requested a way to include sidebars, related content boxes, or other recirculation links within the body of an article. A curated solution -- where users have reasonable control over the presentation and selection of the items on an individual -- is highly-desirable.
+
+Existing solutions are either too generic (`element_group`, `story`) to interpret well in a UX, or too limited (`related_content`, `promo_items`) to be useful in this capacity, or cannot be meaningfully extended in a non-breaking way.
+
+
+## Proposal
+
+Add a new content element `links_list` that contains a narrowly-targeted set of functionality around inline recirculation boxes. Applicable to stories only.
+
+A links_list may have:
+* A plaintext title
+* A subtype (selected from an enumerated-list configured on a per-organization basis)
+* A sequence of up to 10 and at least one `links_list_item`s, each of which has:
+    * A subtype
+    * A plaintext title (required)
+    * A link (required)
+    * A description (ANS `text` element)
+    * An image (ANS `image` or `reference` to an image)
+
+
+## Example
+
+```json
+{
+  "type": "story",
+  "version": "0.10.x",
+
+  "headlines": {
+    "basic": "Man Bites Dog"
+  },
+
+  "content_elements": [
+    {
+      "type": "links_list",
+
+      "subtype": "sidebar",
+      "title": "More 'Biting' Stories",
+
+      "items": [
+        {
+          "type": "links_list_item",
+          "subtype": "blue-label",
+
+          "title": "Lorem ipsum",
+          "link": "http://www.google.com",
```

Comment:
From the last email for Prisa, looks like they need a way to configure additional data for the link: rel="nofollow" and target="_blank". Do u think we should have additional_propertties in links_list_item for custom requirements or introduce specific properties for the needs?

Reply:
I think they should key off of subtype if they really need it.

Reply:
It seems it really depends on how many parameters users want to control. For instance, for RED/BLUE/BLACK label color and nofollow/blank settings, we will have to provide 12 subtypes.

Reply:
Are you suggesting this be a text item which can have an `<a>` tag? I think that opens this up far more than we want to.

Reply:
Do u mean they have to define subtypes something like these:
- blue-label-nofollow
- blue-label
- red-label-nofollow
- red-label
- green-label-nofollow
- green-label
- ...
It looks like more complex and each org might implement the pattern differently which will make Ellipsis hard to standardize the UI.

Reply:
@chadmoone No tags. The entire entry is a link to the URL specified.

Reply:
@chenghsinw @yury-sannikov We should not assume a specific rendering. We're trying to capture a general version of the use case. The front-end can key off of the subtype to determine labels, colors, nofollow, whatever other presentation options they want.  If there is a proven widespread demand for labels or nofollow, specifically, we can add those options at a later time.